src/wattsi.pas

```diff
@@ -48,6 +48,8 @@
    Quiet: Boolean = false;
    Version: Word = (*$I version.inc *); // unsigned integer from 0 .. 65535
    OutputDirectory: AnsiString;
+   SearchIndexJsonFile: Text;
```

Comment:
Nit: maybe this should be named "SearchIndexJsonText"? "File" to me sounds like a filename.